plutus-tx/testlib/PlutusTx/Test/Run/Code.hs

```diff
@@ -64,3 +67,10 @@ assertEvaluatesWithError code = do
             , "Evaluation traces:"
             , Text.unlines evalResultTraces
             ]
+
+assertResultTrue :: TestName -> CompiledCode Bool -> TestNested
+assertResultTrue name code =
```

Comment:
Looks useful! I suggest to drop the `True` suffix like all `asserts` [out there](https://hoogle.haskell.org/?hoogle=assert) do.
```suggestion
assertResult :: TestName -> CompiledCode Bool -> TestNested
assertResult name code =
```